app/services/analytics.rb

```diff
@@ -76,6 +76,7 @@ def browser_attributes
   ADD_EMAIL_VISIT = 'Add Email: enter email visited'.freeze
   AUTHENTICATION_CONFIRMATION = 'Authentication Confirmation'.freeze
   CAC_PROOFING = 'CAC Proofing'.freeze # visited or submitted is appended
+  CAC_PROOFING_SUBMITTED = 'CAC Proofing submitted'.freeze
```

Comment:
oh I just realized the comment on the previous line says "visited or submitted is appended" --- it's maybe not worth changing that now? we could just go in and change them all later?

Reply:
I forgot I had a helper `analytics_submitted` in the FlowStateMachine.  It autogenerates the tokens from the base id for each step.